README.en.md

```diff
@@ -23,12 +23,32 @@ For readability:
   and are either used for testing or for developing and validating what's in assets/.
 
 
-## Development
+## Automated Checks and Testing
 
-This repository uses pre-commit to validate content.
-An integrated testing environment to reproduce the CI pipeline
-is available via docker-compose, which goes on thru a set of steps.
+Below are described the procedures for automated checks and testing implemented, essential for ensuring the quality and integrity of the repository content.
+
+### Automated Checks (Pre-commit)
+
+This repository implements automated checks using [pre-commit](https://pre-commit.com/). The specifications of the checks are defined in the file [`.pre-commit-config.yaml`](.pre-commit-config.yaml).
+
```

Comment:
OK, but use shorter lines. This reduces the PR size when changing documentation files :)